src/data/structure/Recommendation.java

```diff
@@ -0,0 +1,33 @@
+package data.structure;
+
+/**
+ * Represents a recommendation based on a UserProfile with its associated Preferences
+ *    (i.e. the user's preferences).  Contains the Preference that is most correlated
+ *      with the user's preferences as well as total correlation score.
+ * @author benjaminposerow
+ *
+ */
+public class Recommendation {
```

Comment:
I agree.  I usually put them in, I didn't make any explicit choice to exclude them, I just forgot to put them in.  Added them now.  You don't just need them for HashMaps or Collections, you might also need them in conditional logic, testing code, etc.  The only point I was making was the code I had written up to that point hadn't made it necessary, but I really should have added them.